Assets/ModScripts/Submodules/DerangedKeypad.cs

```diff
@@ -84,14 +85,18 @@ public override void OnButtonPress()
         if (Module.IsModuleSolved())
             return;
 
-        if (!Module.IsSolving() && !Module.CheckValidComponents())
+        if (!Module.IsSolving())
         {
-            Debug.LogFormat("[The Cruel Modkit #{0}] Strike! The button was pressed when the component selection was [{1}] instead of [{2}].", ModuleID, Module.GetOnComponents(), Module.GetTargetComponents());
-            Module.CauseStrike();
-            return;
+            if (!Module.IsSolving() && !Module.CheckValidComponents())
```

Comment:
```suggestion
            if (!Module.CheckValidComponents())
```
Forgot to delete this?